docs/framework/configure-apps/file-schema/wcf/bindings.md

```diff
@@ -1,28 +1,32 @@
 ---
-title: "<bindings>"
-ms.date: "03/30/2017"
+title: "&lt;bindings&gt;"
```

Comment:
This is an issue for Loc: https://github.com/dotnet/docs/pull/10137

Why do we need to roll back this change?

Reply:
When it was "<bindings>", the text was removed in the build process.

I'll try the "\<bindings\>"

Reply:
It seems to be working fine as it is right now on docs: https://docs.microsoft.com/en-us/dotnet/framework/misc/binding

![image](https://user-images.githubusercontent.com/12971179/53834221-4edb5f80-3f3f-11e9-8392-9e11415dd5fe.png)

Reply:
I'll update to match that branch